Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM quay.io/keboola/docker-custom-python:2.0.4
 
-RUN mkdir -p /var/www && chown www-data:www-data /var/www
+# Create director for user packages
+# This directory is usually created automatically by pip
+# ... but if it doesn't exist when you run the script
+# ... then the modules installed during the transformation are not loaded automatically!
+# ... because the loader thinks that this directory does not exist (it did not exist at the start of the script)
+RUN mkdir -p /var/www/.local/lib/python3.8/site-packages
```

Comment:
i think we should probably add here a note to change this if/when we change the python version during next update 
![image](https://user-images.githubusercontent.com/569139/96597823-8eaab280-12ee-11eb-8442-bf2490453912.png)
https://keboola.atlassian.net/wiki/spaces/SUP/pages/72024277/Python+R+Julia+Updates

Reply:
very good note, thank you! ...  I'd rather get that path dynamically by `python -c "import site; print(site.USER_SITE)"`

Reply:
yes, that would be ideal